__tests__/app.test.js

```diff
@@ -249,3 +249,30 @@ describe('GET /api/users', () => {
         })
     });
 });
+
+describe('DELETE /api/comments/:comment_id', () => {
+    it('returns status of 204 when sent the correct id and comment is removed from the comments table', () => {
+        return request(app)
```

Comment:
Please also check that the comment has actually been deleted from the database by making a request to your GET commentsById endpoint and asserting that you expect the response to be 404 comment not found after doing the patch.